### ISSUE TITLE
Unexpected behavior when using things such as empty string

### ISSUE THREAD

**Issue by buzzsawddog** [tools4j/spockito#1]
I am in the process of converting some unit tests that I have to use spockito unroll.  I have to say I am glad I finally found this :-D  But I am running into some issues and it may be just lack of familiarity with the tool.

This is what I am switching to:

```
  @Test(expected = InputViolationException.class)
  @Spockito.Unroll({
    "|Input|",
    "||",
    "|a|",
    "|select|",
    "|'|",
    "| |"
  })
  public void testValidateString(String input) {
    ParamaterUtil.validateString(input);
  }
```

This is what I am switching from:

```
  @Test(expected = InputViolationException.class)
  public void testBlankString() {
    ParamaterUtil.validateString("");
  }

  @Test(expected = InputViolationException.class)
  public void testContainsLetter() {
    ParamaterUtil.validateString("a");
  }

  @Test(expected = InputViolationException.class)
  public void testContainsString() {
    ParamaterUtil.validateString("select");
  }

  @Test(expected = InputViolationException.class)
  public void testContainsSingleQuote() {
    ParamaterUtil.validateString("'");
  }

  @Test(expected = InputViolationException.class)
  public void testContainsSpace() {
    ParamaterUtil.validateString(" ");
  }
```

The problem I am running into is with empty string and blank string.  The only tests that run are a, select and single quote.  The other ones just dont even seem to show.

**Comment by terzerm** [comment]
Thanks for your interest in spockito!

Most of the cases that you describe here are possible, but you have also encountered a small bug for the case of an empty string: If all strings in a table row are empty, the row is ignored.

I will fix the bug in version 1.3 but fortunately there is also a workaround. You can add a second column with a non-empty value such as a test description. Your table would then look like this:

```java
    @Test(expected = InputViolationException.class)
    @Spockito.Unroll({
            "|Input|Description|",
            "|''|blank string|",
            "|a|letter|",
            "|select|string|",
            "|'|single quote|",
            "|' '|space|"
    })
    public void testValidateString(String input) {
        ParamaterUtil.validateString(input);
    }
```

**Comment by terzerm** [comment]
Fixed as of version 1.3

**Comment by buzzsawddog** [comment]
Thanks I appreciate this.
